docs/ruleset_functions/get_building_segment_window_wall_areas.md

```diff
@@ -41,8 +42,8 @@ Logic:
 
```

Comment:
for line 35, please change the note from "If surface is above-grade wall and adjacent to the exterior" to "If surface is above-grade wall and is part of exterior or semi-exterior building envelope"

Reply:
Updated.